bugsnag-android-core/src/main/java/com/bugsnag/android/internal/InternalMetricsNoop.kt

```diff
@@ -0,0 +1,19 @@
+package com.bugsnag.android.internal
+
+class InternalMetricsNoop : InternalMetrics {
+    override fun toJsonableMap(): Map<String, Any> {
+        return mapOf()
+    }
+
+    override fun setConfigDifferences(differences: Map<String, Any>) {
+    }
+
+    override fun setCallbackCounts(newCallbackCounts: Map<String, Int>) {
+    }
+
+    override fun notifyAddCallback(callback: String) {
+    }
+
+    override fun notifyRemoveCallback(callback: String) {
+    }
```

Comment:
Kotlin has a nice short-hand syntax for these sorts of things:

```suggestion
object InternalMetricsNoop : InternalMetrics {
    override fun toJsonableMap(): Map<String, Any> = emptyMap()
    override fun setConfigDifferences(differences: Map<String, Any>) = Unit
    override fun setCallbackCounts(newCallbackCounts: Map<String, Int>) = Unit
    override fun notifyAddCallback(callback: String) = Unit
    override fun notifyRemoveCallback(callback: String) = Unit
```